### ISSUE TITLE
The file /tests/data/pe_mingw is detected as malware by Windows Defender and several other vendors

### ISSUE THREAD

**Issue by germanlancioni** [VirusTotal/yara#1655]
I'm wondering if this is a false positive or something malicious indeed went into this file recently (last modif: Sept 17 2021). When downloading previous versions of Yara, there were no malware detections. However, when downloading the latest release candidate (or cloning the repository), the file "pe_mingw" is detected as malware by either Microsoft Windows Defender or other cybersecurity vendors.

Tested with: [YARA v4.2.0-rc1](https://github.com/VirusTotal/yara/releases/tag/v4.2.0-rc1) and by cloning the repository as of today Feb 24, 2022.

**Comment by plusvic** [comment]
`pe_mingw` is a simple "Hello world!" program that does nothing, but it seems that some antiviruses decided to detect it for some reason. This file was added relatively recently as data for test cases.

**Comment by germanlancioni** [comment]
Thanks, I imagined that's the case. However, I'm thinking that even if it's a false positive, is somehow harming the "adoption" of Yara. If you try to clone or download the repo, you get stopped by Defender or others, which is likely to steer people away from the library.
I tried to figure out why it's detected but a diff is hard to follow given it's binary data. We know that the file only started to get detected since the last update on Sept 17 2021 (\N separators added?).

Is there something we can do to revert it back to "not being detected", assuming we are 100% sure this is not malware?

**Comment by plusvic** [comment]
I'm afraid the only options here are creating a new file that can be used with the test case (and hope that this time antiviruses don't detect it) or removing the test case altogether.
